script/install-cni.sh

```diff
@@ -103,7 +103,7 @@ EOF
   "kubernetes": {
     "kubeconfig": "${WHEREABOUTS_KUBECONFIG_LITERAL}"
   },
-  "reconciler_cron_expression": "30 4 * * *"
+  "reconciler_cron_expression": "${WHEREABOUTS_RECONCILER_CRON:-30 4 * * *}"
```

Comment:
nit: I would probably initialize this in the [beginning of the file](https://github.com/k8snetworkplumbingwg/whereabouts/blob/6a7b22ccc468a114157b97c8ce21ea49781c2d0c/script/install-cni.sh#L29), and just use it here. I feel that's more inline with the rest of the code.

Reply:
(optional; you can leave it as is).